lib/Db/MessageMapper.php

```diff
@@ -322,6 +322,16 @@ public function findUidsByQuery(Mailbox $mailbox, SearchQuery $query, ?int $limi
 			);
 		}
 
+		if ($query->getSubject() !== null) {
+			$select->andWhere(
+				$qb->expr()->iLike(
+					'subject',
+					$qb->createNamedParameter('%' . $query->getSubject() . '%', IQueryBuilder::PARAM_STR),
```

Comment:
Just thinking... should subject be escaped?

Reply:
I guess. But I naively assumed that our db abstraction does that with every parameter. Is that not the case? Checking …

Reply:
Well depends, do you want people to be able to use placeholders?
Should they search for "he_lo" or "healo", "heblo", ... when they type "he_lo"?
If the first one, you need to escape, if the second one, dont